typings/jsonwebtoken/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'jsonwebtoken';
```

Comment:
https://www.npmjs.com/package/@types/jsonwebtoken
Could we use this types?

Reply:
Before adding it explicitly to types, I used the above as well. Didn't work for me.